### PR TITLE
add params example with toml parameters file

### PR DESCRIPTION
**Pull request by dtrifiro**
Related to https://github.com/iterative/dvc/pull/4258

### CODE REVIEW

File: content/docs/command-reference/params/index.md [hunk 1]
```diff
@@ -24,8 +24,8 @@ dependencies: _parameters_. Parameters are defined using the the `-p`
 
 In contrast to a regular <abbr>dependency</abbr>, a parameter is not a file (or
 directory). Instead, it consists of a _parameter name_ (or key) to find inside a
-YAML or JSON _parameters file_. Multiple parameter dependencies can be specified
-from one or more parameters files.
+YAML, JSON or TOML _parameters file_. Multiple parameter dependencies can be
```
Comment (jorgeorpinel) [inline content/docs/command-reference/params/index.md:27]:
```suggestion
YAML, JSON, or TOML _parameters file_. Multiple parameter dependencies can be
```

File: content/docs/command-reference/params/index.md [hunk 2]
```diff
@@ -143,6 +143,14 @@ $ dvc run -n train -d logs/ -o users.csv \
...
 ```
 
+Alternatively, JSON or TOML can be used to specify the parameters file:
+
+```dvc
+$ dvc run -n train -d logs/ -o users.csv \
+          -p params.toml:threshold,classes_num \
+          python train.py
+```
+
```
Comment (jorgeorpinel) [inline content/docs/command-reference/params/index.md:153]: Thanks but probably not needed, it's repetitive and we already have a note reminding the user that JSON and TOML are also supported. So:
```suggestion
```

**Comment by jorgeorpinel** [comment]
Also, please let is know once iterative/dvc/pull/4258 is merged. Thanks!

**Comment by dtrifiro** [comment]
> Also, please let is know once [iterative/dvc/pull/4258](https://github.com/iterative/dvc/pull/4258) is merged. Thanks!

It has been merged: https://github.com/iterative/dvc/pull/4258

**Comment by shcheklein** [comment]
Thanks @dtrifiro !

**Pull request merged**